notification-server/src/processor.rs

```diff
@@ -0,0 +1,78 @@
+use concordium_rust_sdk::{
+    cis2,
+    cis2::Event,
+    types,
+    types::{
+        AccountTransactionEffects, BlockItemSummaryDetails::AccountTransaction,
+        ContractTraceElement,
+    },
+};
+use concordium_rust_sdk::base::contracts_common::AccountAddress;
+use concordium_rust_sdk::types::Address::Account;
+use futures::{Stream, StreamExt};
+
+fn get_cis2_events_addresses(effects: &AccountTransactionEffects) -> Option<Vec<AccountAddress>> {
+    match &effects {
+        AccountTransactionEffects::ContractUpdateIssued { effects } => Some(
+            effects
+                .iter()
+                .flat_map(|effect| match effect {
+                    ContractTraceElement::Updated { data } => data
+                        .events
+                        .iter()
+                        .map(|event| match cis2::Event::try_from(event) {
+                            Ok(Event::Transfer { to, .. }) => Some(to),
+                            Ok(Event::Mint { owner, .. }) => Some(owner),
```

Comment:
I had some time thinking about the `Burn` event and I think we should add it since the `operator` can also burn your tokens on your behalf so being notified if your tokens get burned might be beneficial since the owner might not initiate the action.